cli/src/commands/withdraw_auction_house.rs

```diff
@@ -47,9 +47,17 @@ pub fn process_withdraw_auction_house_treasury(
 
     let Mint { decimals, .. } = Mint::unpack(&token_mint_data[..])?;
 
-    let amount_to_withdraw_with_decimals =
+    let mut amount_to_withdraw_with_decimals =
         amount.saturating_mul(10u64.saturating_pow(decimals.into()));
 
+    if treasury_mint.eq(&spl_token::native_mint::id()) {
+        info!("Deducting the rent amount from the withdrawal value");
+        let rent_exemption_lamports = client.get_minimum_balance_for_rent_exemption(0)?;
+
+        amount_to_withdraw_with_decimals =
+            amount_to_withdraw_with_decimals - rent_exemption_lamports;
+    }
+
```

Comment:
Don't always have to deduct rent_exemption only if the amount would leave the treasury under rent exemption.